instrumentation/java-http-client/library/src/main/java/io/opentelemetry/instrumentation/httpclient/JavaHttpClientTelemetry.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.httpclient;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.httpclient.internal.HttpHeadersSetter;
+import io.opentelemetry.instrumentation.httpclient.internal.OpenTelemetryClient;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+/** Entrypoint for instrumenting Java HTTP Client. */
+public final class JavaHttpClientTelemetry {
```

Comment:
I named the public api classes `JavaHttpClient..` but some of the other classes still use `JdkHttpClient...`. I think we should choose how we want to call this and name all the classes in the same way.

Reply:
+1 on renaming everything from `Jdk*` to `Java*`